Sources/Datadog/RUM/DataModels/RUMDataModelsMapping.swift

```diff
@@ -14,6 +14,15 @@ extension Int {
     }
 }
 
+extension UInt {
```

Comment:
we can implement `toInt64` in `extension BinaryInteger`, it should cover all our cases

Reply:
Could be 👍. I will incorporate next upcoming PR (too much rebasing to do it here).